mungegithub/mungers/inactive-review-handler.go

```diff
@@ -0,0 +1,168 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mungers
+
+import (
+	"fmt"
+	"github.com/golang/glog"
+	githubapi "github.com/google/go-github/github"
+	"github.com/spf13/cobra"
+	"k8s.io/test-infra/mungegithub/features"
+	"k8s.io/test-infra/mungegithub/github"
+	"time"
+)
+
+type InactiveReviewHandler struct {
+	features *features.Features
+}
+
+func init() {
+	h := &InactiveReviewHandler{}
+	RegisterMungerOrDie(h)
+}
+
+// Name is the name usable in --pr-mungers
+func (i *InactiveReviewHandler) Name() string { return "inactive-review-handler" }
+
+// RequiredFeatures is a slice of 'features' that must be provided
+func (i *InactiveReviewHandler) RequiredFeatures() []string {
+	return []string{features.RepoFeatureName, features.AliasesFeature}
+}
+
+// Initialize will initialize the munger
+func (i *InactiveReviewHandler) Initialize(config *github.Config, features *features.Features) error {
+	i.features = features
+	return nil
+}
+
+// EachLoop is called at the start of every munge loop
+func (i *InactiveReviewHandler) EachLoop() error { return nil }
+
+// AddFlags will add any request flags to the cobra `cmd`
+func (i *InactiveReviewHandler) AddFlags(cmd *cobra.Command, config *github.Config) {}
+
+// Determine whether the PR is active based on number of seconds
+// A PR is inactive if it is not being modified for more than one week
+func (i *InactiveReviewHandler) isPRActive(issueCreatedAt *time.Time, comments []*githubapi.IssueComment, reviewComments []*githubapi.PullRequestComment) bool {
```

Comment:
I don't understand how this function is better than the version you just removed? It has twice the amount of lines, it's more complex to understand (the semantic isn't as good), and does repeat some constant and logic multiple times (`time.Since... <= 7*24`)